cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -46,33 +46,42 @@ data:
     # CRI Log Example:
     # 2016-02-17T00:04:05.931087621Z stdout F [info:2016-02-16T16:04:05.930-08:00] Some log text here
     <source>
-      type tail
+      @type tail
       path /var/log/containers/*.log
       pos_file /var/log/gcp-containers.log.pos
       tag reform.*
       read_from_head true
-      format multi_format
-      <pattern>
-        format json
-        time_key time
-        time_format %Y-%m-%dT%H:%M:%S.%NZ
-      </pattern>
-      <pattern>
-        format /^(?<time>.+) (?<stream>stdout|stderr) [^ ]* (?<log>.*)$/
-        time_format %Y-%m-%dT%H:%M:%S.%N%:z
-      </pattern>
+      format none
     </source>
 
     <filter reform.**>
-      type parser
+      @type parser
```

Comment:
Is it required to split this parser into an individual section? I recall that we changed this when debugging a separate issue. Is this the culprit though?

If we do need to change it, maybe reply to this thread to clarify the reason?

Reply:
I'm pretty confident using this as a filter plugin squelched the logs that were being emitted to invalid formats. However, I don't see that I have `emit_invalid_record_to_error` in this section, so now I'm a little confused. I'll test locally and try to get a better story around this.

Reply:
Thanks for checking!

Reply:
Just confirmed, the previous way of incorporating format multi_format into the source causes the plugin to emit an error complaining about not being able to parse the format, which gets directed to stdout/stderr, which causes an infinite loop of not being able to parse logs from the containers directory. The change as we have it ensures that that parse failure does not get emitted, and it just gracefully exhausts each format in the multi format plugin.

Reply:
Sounds good. Thanks!